net/gensync/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            nislab gensync 1.0.4 v
+git.branch              vanilla
+
+version                 1.0.4
```

Comment:
I apologize - I should have also mentioned that `version` is also being implicitly set by `github.setup`; this line is redundant and may be removed.